json_templates/api_tokens.json

```diff
@@ -0,0 +1,18 @@
+{
+  "tokens ": {
+    "this websites api tokens along with the IP from which they were created (used when resetting token)": {
+      "{users:IP:address}": "{token}"
```

Comment:
Shouldn't here template be like this: {users:IPaddress} ? or address is a different value ?

Reply:
I tried to convey that IP is saved using ':' as divider between numbers.
It is now changed for a more clear phrase.